reporting/api/src/main/java/org/jboss/windup/reporting/model/InlineHintModel.java

```diff
@@ -17,7 +17,7 @@
 @TypeValue(InlineHintModel.TYPE)
 public interface InlineHintModel extends EffortReportModel, FileLocationModel, TaggableModel
 {
-    String TYPE = "Hint";
+    String TYPE = "InlineHintModel";
```

Comment:
Same here. If there is 10000 hints, this adds 110 kb to the JSON data. Not sure if we care.